Upgrade from `android.support.v4.os.ParcelableCompat`

The androidx.core.os.ParcelableCompat class is deprecated (as of Android API 26). To migrate our code, we should use Parcelable.ClassLoaderCreator directly instead of ParcelableCompat.

Migrate from `android.support` to `androidx` package.

```
> .\gradlew clean build
```

```
> Task :app:compileDebugJavaWithJavac
android_packages_apps_Calculator\app\src\main\java\com\android\calculator2\viewpager\VerticalViewPager.java:43: error: cannot find symbol
import android.support.v4.os.ParcelableCompat;
                            ^
  symbol:   class ParcelableCompat
  location: package android.support.v4.os

```

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -38,5 +38,6 @@ dependencies {
 
     implementation 'androidx.appcompat:appcompat:1.7.0'
     implementation 'androidx.recyclerview:recyclerview:1.4.0'
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
     implementation 'ai.elimu:model:model-2.0.100'
 }

--- a/app/src/main/java/com/android/calculator2/viewpager/VerticalViewPager.java
+++ b/app/src/main/java/com/android/calculator2/viewpager/VerticalViewPager.java
@@ -40,10 +40,9 @@ import android.os.Bundle;
 import android.os.Parcel;
 import android.os.Parcelable;
 import android.os.SystemClock;
-import android.support.v4.os.ParcelableCompat;
-import android.support.v4.os.ParcelableCompatCreatorCallbacks;
+import androidx.core.os.ParcelableCompat;
+import androidx.core.os.ParcelableCompatCreatorCallbacks;
 import androidx.core.view.AccessibilityDelegateCompat;
-import android.support.v4.view.KeyEventCompat;
 import androidx.core.view.MotionEventCompat;
 import androidx.core.view.VelocityTrackerCompat;
 import androidx.core.view.ViewCompat;
@@ -2453,9 +2452,9 @@ public class VerticalViewPager extends ViewGroup {
                     if (Build.VERSION.SDK_INT >= 11) {
                         // The focus finder had a bug handling FOCUS_FORWARD and FOCUS_BACKWARD
                         // before Android 3.0. Ignore the tab key on those devices.
-                        if (KeyEventCompat.hasNoModifiers(event)) {
+                        if (event.hasNoModifiers()) {
                             handled = arrowScroll(FOCUS_FORWARD);
-                        } else if (KeyEventCompat.hasModifiers(event, KeyEvent.META_SHIFT_ON)) {
+                        } else if (event.hasModifiers(KeyEvent.META_SHIFT_ON)) {
                             handled = arrowScroll(FOCUS_BACKWARD);
                         }
                     }

